sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/strategy/CanaryStrategy.java

```diff
@@ -130,6 +130,11 @@ private Step getNextCanaryStep() {
         return null;
     }
 
+    @Override
+    public String toString() {
```

Comment:
How about a separate `getName()` function against the Strategy interface? That'd make it clearly required by any implementors, whereas a custom implementation of `toString()` could be missed. `toString()` could also someday be needed for returning debug/logging info about the strategy that wouldn't be relevant to end users.

Reply:
Yeah, I think that sounds reasonable, will update.